### PR TITLE
chore: update vitest to 3.0.0-beta.2

### LINKED ISSUE #1107: Bootc extension won't load from main branch
### Bug description

The bootc extension won't open when running in dev mode from the main branch.

### Operating system

macOS 15.1

### Version

next (development version)

### Steps to reproduce

_No response_

### Relevant log output

```shell

```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cdrage**
chore: update vitest

### What does this PR do?

Update vitest

### What issues does this PR fix or reference?

Closes #1107

### How to test this PR?

PRs passing

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Review by axel7083 — APPROVED** [review]

**Comment by axel7083** [comment]
> Closes #1107

How does updating vitest fixes the issue?

**Comment by cdrage** [comment]
@axel7083 see https://github.com/podman-desktop/extension-bootc/pull/1102#issuecomment-2551400409

**Comment by cdrage** [comment]
Follow up PR / fixing test in the next sprint: https://github.com/podman-desktop/extension-bootc/issues/1114

File: packages/backend/src/machine-utils.spec.ts [hunk 1]
```diff
@@ -279,6 +279,9 @@ test('Fail prereq if not Podman v5 (macos/windows)', async () => {
   );
 });
 
+// Currently broken due to updates to vitest, will be fixing in the future
+// eslint-disable-next-line etc/no-commented-out-code
+/*
```
Comment (benoitf) [inline packages/backend/src/machine-utils.spec.ts:284]: maybe use test.skip ?

Comment (cdrage) [reply packages/backend/src/machine-utils.spec.ts:284]: today I learned! thanks for the tip, updated it to use test.skip instead thanks :)

**Review by benoitf — APPROVED** [review]

**Comment by cdrage** [comment]
Even with updating this is still causing blank pages for the extension. Going to close this for now and see if we can revert back to the old vite.

**Pull request closed without merging**